rs/bitcoin/kyt/src/providers.rs

```diff
@@ -143,6 +156,26 @@ fn make_request(
     }
 }
 
+fn make_post_request(
+    json_rpc_url: &str,
+    txid: Txid,
+    max_response_bytes: u32,
+) -> Result<CanisterHttpRequestArgument, String> {
+    let (url, header) = parse_authorization_header_from_url(json_rpc_url)?;
```

Comment:
@ninegua : maybe a dumb question but why are the credentials actually passed by URL when at the end they will be sent by header? Also asking because some characters cannot be used in URLs.

Reply:
The bitcoind RPC only takes authorization from headers. I'm just using a single URL string to simplify the configuration, which should be sufficient for testing purpose. This can be improved in the future if better support is needed.